### PR TITLE
user-service setup

### PR DESCRIPTION
**Pull request by d4rckh**
Added spring boot and spring boot web depency versions to pom.xml and created user-service

### CODE REVIEW

File: user-service/pom.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    </dependencies>
+
+    <properties>
+        <maven.compiler.source>22</maven.compiler.source>
+        <maven.compiler.target>22</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <build>
+        <plugins>
```
Comment (MihaiCatalin97) [inline user-service/pom.xml:32]: Let's use pluginManagement in parent as well :D

File: pom.xml [hunk 2]
```diff
@@ -26,6 +27,16 @@
                 <artifactId>lombok</artifactId>
                 <version>1.18.34</version>
             </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot</artifactId>
+                <version>3.3.2</version>
```
Comment (MihaiCatalin97) [inline pom.xml:33]: Some versions generally need to be linked together, just like these two

In this case we use properties

Up above the dependencies you can declare some properties:

<properties>
    <spring.version>3.3.2</spring.version>
</properties>

and then use a placeholder here

<version>${spring.version}</version>

**Comment by MihaiCatalin97** [comment]
Please set a more explicit title on this PR
Something like "User Service setup"

**Review by MihaiCatalin97 — APPROVED** [review]

**Review by Sorana09 — APPROVED** [review]

**Comment by MihaiCatalin97** [comment]
image

**Pull request merged**